spec/models/concerns/push_updates_for_offer_spec.rb

```diff
@@ -0,0 +1,43 @@
+# require 'rails_helper'
```

Comment:
This looks odd. Was it a debug that got pushed by mistake ? : )

Reply:
oops... didn't mean to include this file!